app/views/stories/tagged_articles/_sidebar.html.erb

```diff
@@ -3,7 +3,6 @@
     <div class="sidebar-bg" id="sidebar-bg-left"></div>
     <aside class="side-bar">
       <% if @tag && @tag.rules_html.present? %>
-        <div class="widget">
         <%= application_policy_content_tag("div", record: Article, query: :create?, class: "widget") do %>
```

Comment:
looking at the recent commit history, this policy content tag had been guarded by a feature flag previously, and so the issue wasn't noticed immediately.